frontend/src/app/services.ts

```diff
@@ -75,22 +75,30 @@ const getFetchInit = () => {
     };
 };
 
-export const downloadApartmentUnconfirmedMaximumPricePDF = (apartment: IApartmentDetails, additionalInfo?: string) => {
+export const downloadApartmentUnconfirmedMaximumPricePDF = (
+    apartment: IApartmentDetails,
+    additionalInfo?: string,
+    requestDate?: string
```

Comment:
The requestDate is always required, so the question mark can be removed from the type hint (here and below also) 😉